src/main/java/com/github/susom/dbgoodies/etl/Etl.java

```diff
@@ -272,36 +272,35 @@ SaveAsAvro fetchSize(int nbrRows) {
     }
 
     /**
-     *  create a BigQueryWriter and hook up with Database select callback. when query finish, signal writer to turn off workers
-     *
-     *  the writer uses BigQuery stream api, which requires unique object id for each row. EntryIdFields is important, you should use primary key field list for this. BigQueryWriter will take this comma separated string input and compute object id for each row when upload to BigQuery
-     *
-     *  requres Google Client Credential file properly set up in env
+     * create a BigQueryWriter and hook up with Database select callback. when query finish, signal writer to turn off workers
+     * <p>
+     * the writer uses BigQuery stream api, which requires unique object id for each row. EntryIdFields is important, you should use primary key field list for this. BigQueryWriter will take this comma separated string input and compute object id for each row when upload to BigQuery
+     * <p>
+     * requres Google Client Credential file properly set up in env
      */
     public void start() throws Exception {
-        Optional<String> google_credential_file = Optional.ofNullable(System.getenv("GOOGLE_APPLICATION_CREDENTIALS")) ;
-        if(!google_credential_file.isPresent()){
-            log.error("Can not get google api credential file from environment variable GOOGLE_APPLICATION_CREDENTIALS ");
-            System.exit(1);
-        }
+      Map<String, String> labels = new HashMap<>();
+
+      Optional<String> google_credential_file = Optional.ofNullable(System.getenv("GOOGLE_APPLICATION_CREDENTIALS")) ;
+
+      labels.put("job", "testing");
```

Comment:
Probably shouldn't be hard-coding the labels

Reply:
I didn’t make any Avro changes... probably Idea auto-formatting the code. The spacing was all over the place.

Reply:
If you go into the diff settings above and check "ignore whitespace" you will see it's all whitespace... I try to catch that before I commit but hitting cmd-option-L is practically a nervous tick of mine. Do we have a coding standard somewhere? eg. 2 vs 4 spaces for indentation....

Reply:
removed line 286 and committed to this pr